### PR TITLE
Add tooltips to channel tags shown in devhub

### LINKED ISSUE #4398: the (AMO) and (Self) tags aren't localised
We show (AMO) and (Self) next to versions in a few places if an addon has both unlisted and listed versions.  But the text isn't localised.

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #4398

### CODE REVIEW

**Comment by eviljeff** [comment]
That's why I didn't request review yet 😛

**Comment by eviljeff** [comment]
now with l10n!

**Comment by eviljeff** [comment]
@EnTeQuAk can look (and weep) at the tooltip I added to his clean devhub landing page.

File: src/olympia/devhub/templates/devhub/includes/addon_details.html [hunk 1]
```diff
@@ -55,7 +55,9 @@
 {% if addon.has_listed_versions() %}
     {% if addon.has_unlisted_versions() %}
       <li class="distribution-tag-listed">
-        <span class="distribution-tag-listed">AMO</span>
+        <span class="distribution-tag-listed tooltip"
+              title="{{ _('Listed on this site after passing code review. Automatic updates are handled by this site.') }}"
```
Comment (EnTeQuAk) [inline src/olympia/devhub/templates/devhub/includes/addon_details.html]: That's one too many "this site" for my taste but maybe that's just me.

**Comment by EnTeQuAk** [comment]
screenshot from 2017-01-24 08-06-24

Just adding a screenshot how this looks on the new pages. I don't know about the colors but they're unified across the page for now and it doesn't look too itchy to me so let's do it!

File: src/olympia/devhub/templates/devhub/includes/addon_details.html (hunk 1 above)
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/includes/addon_details.html]: heh, you've got a point, but: https://github.com/mozilla/addons-server/issues/4398#issuecomment-274137542

**Comment by eviljeff** [comment]
Yeah, the style of the tooltips could benefit from an update - and to not depend on an impala stylesheet file.

**Pull request merged**